setup.cfg

```diff
@@ -80,7 +79,7 @@ tls =
 
 conch =
     pyasn1
-    cryptography >= 2.6
+    cryptography >= 2.6, < 37.0
```

Comment:
Why?

Reply:
Thanks @alex  for the review. Our tests are failing with the latest cryptography 

It might be because the tests are not well written for openssl 3 

But meanwhile we investigate why the tests are failing, I think that we should have a green trunk.

You can see the errors here  trunk was green at merge time, and then after a re-run tests are failing as the latest crytpography is used.

https://github.com/twisted/twisted/runs/6178994160?check_suite_focus=true

Reply:
Ok. FWIW I'd suggest this be in it's own PR, as it's not related to dropping 36 support.

I'd also encourage folks to get cryptography 37 working soon, as next week there'll be an openssl security release and cryptography 37.0.1 will be necessary for wheels to get a security update.